layouts/DocsPage/DocsPage.tsx

```diff
@@ -39,11 +39,19 @@ const DocsPage = ({
   children,
 }: DocsPageProps) => {
   const route = useCurrentHref();
+  const currentPage = getCurrentPageWithScope(route);
   const { setVersions } = useContext(DocsContext);
+  const [articleList, setArticleList] = useState<LinkWithRedirectList>({});
 
   const { current, latest, available } = versions;
 
   useEffect(() => {
+    async function getRedirectsMap() {
```

Comment:
I think we download it in versions too. Let's move it to separate hook and add caching to it, so we don't download it on every page load.